rpm/BerlinVegan.yaml

```diff
@@ -1,12 +1,12 @@
 Name: harbour-berlin-vegan
 Summary: Berlin-Vegan guide
-Version: 3.1.1-alpha
```

Comment:
@jmastr note: We need to do something different here: ```-``` characters are not allowed in this file (as this version is used for filenames and it uses ```-```s to separate version from this release string.

Reply:
no problemo, how about `3.2.1a`, `3.2.1alpha`, or `3.2.1+alpha` for example?